src/main/java/seedu/address/logic/commands/editcommands/EditInternshipCommand.java

```diff
@@ -26,6 +29,7 @@
 public class EditInternshipCommand extends EditCommand {
 
     public static final String MESSAGE_SUCCESS = "Edited Internship: %1$s";
+    public static final String DISPLAY_MESSAGE_SUCCESS = "Edited the internship at index %1$s for this company: ";
```

Comment:
Unfortunately due to the feature freeze, I don't think we're allowed to change the UI output :(